tokio/src/runtime/park.rs

```diff
@@ -305,35 +305,55 @@ impl Inner {
         Arc::into_raw(this) as *const ()
     }
 
+    /// # Safety
+    ///
+    /// The pointer must have been created by [`Self::into_raw`].
     unsafe fn from_raw(ptr: *const ()) -> Arc<Inner> {
-        Arc::from_raw(ptr as *const Inner)
+        unsafe { Arc::from_raw(ptr as *const Inner) }
```

Comment:
This function is very short, and it’s clear enough for developers to reason about the safety requirements, so we don’t need additional "Safety" comments here”